operator-common/pom.xml

```diff
@@ -131,6 +135,7 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
```

Comment:
Is this needed? Isn't it specified as `test` in the main pom.xml?

Reply:
True.